tutorials/llm/llama-3/llama3-lora-nemofw.ipynb

```diff
@@ -0,0 +1,587 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d3323204-1463-4df3-8c75-5e95b6d66ba1",
+   "metadata": {},
+   "source": [
+    "# Creating a Llama-3 LoRA adapter with NeMo Framework"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29f3d632-44a0-4e6c-9229-b70bbcff1e99",
+   "metadata": {},
+   "source": [
+    "This notebook showcases performing LoRA PEFT **Llama 3 8B** on [PubMedQA](https://pubmedqa.github.io/) using NeMo Framework. PubMedQA is a Question-Answering dataset for biomedical texts.\n",
+    "\n",
+    "> `NOTE:` Ensure that you run this notebook inside the [NeMo Framework container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo) which has all the required dependencies. Instructions are available in the associated tutorial README."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50de4d53",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install ipywidgets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9211a17d-17a1-4b31-bae3-b3d5f6c4860f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Set your Hugging Face access token\n",
+    "\n",
+    "HF_ACCESS_TOKEN = \"<YOUR_HUGGINGFACE_ACCESS_TOKEN>\""
```

Comment:
FWIW - I did not run into an issue with the original method, but I agree that the `.login()` method will verify your token immediately instead of needing to wait to see potential token issues during following cells.

Reply:
added